pkg/apiclient/project/forwarder_overwrite.go

```diff
@@ -0,0 +1,9 @@
+package project
+
+import (
+	"github.com/argoproj/pkg/grpc/http"
+)
+
+func init() {
+	forward_ProjectService_List_0 = http.UnaryForwarder
+}
```

Comment:
Wonder where is this file used?

Reply:
The init method is executed automatically when `pkg/apiclient/project` package is imported: https://golang.org/doc/effective_go.html#init